apps/patient/src/components/PickupOptions.tsx

```diff
@@ -15,7 +15,7 @@ interface PickupOptionsProps {
   handleSetPreferred: (id: string) => void;
   loadingMore: boolean;
   showingAllPharmacies: boolean;
-  courierEnabled: boolean;
+  showHeading: boolean;
```

Comment:
Why are we changing from the specific `courierEnabled` to the generic `showHeading`? What if we want a different header in the future?

Reply:
This was a bug. If only mail order was enabled, it would not show the header on pickup options, like this...

<img width="360" alt="image" src="https://github.com/Photon-Health/client/assets/3934326/3eb838db-1c45-4abf-a85f-4f7525d63ccf">

Also note, we were piping `enableMailOrder` into the `courierEnabled` prop

<img width="498" alt="image" src="https://github.com/Photon-Health/client/assets/3934326/5bfc7a00-8865-4b3d-8493-35526ce2cf64">

Reply:
So it should be "showDeliveryOptions" is what you're saying?